.github/workflows/build.yml

```diff
@@ -2,6 +2,8 @@ name: Build sub target
 
 on:
   workflow_call:
+    secrets:
+      coverity_api_token:
```

Comment:
I don't think this should be handled here... or I think this may be a security risk? (or it may not be working?)

Need to read github documentation but I think api tokens should be only referenced by using the secrets github way and never passed as an args to other actions. (since they may be leaked in the build logs)